docs/gen_schema.jl

```diff
@@ -60,7 +80,7 @@ end
 
 os_line_separator() = Sys.iswindows() ? "\r\n" : "\n"
 
-function gen_schema(T::DataType, prefix = prefix)
+function gen_schema(T::DataType, prefix = prefix; pandera = true)
```

Comment:
Why is `pandera` optional?

Reply:
Because I hacked the remarks field into each type specifically for pandera. But the generation of the Config doesn't need it.

Frankly, this whole file is a hack, but it works. Doing it right requires a new package that actually implements the whole of JSONSchema.